modular_tannhauser/_skyrat_override/modules/title_screen/code/_title_screen_defines.dm

```diff
@@ -0,0 +1,234 @@
+#define DEFAULT_TITLE_MAP_LOADTIME 150 SECONDS
```

Comment:
do we really need to re-implement the whole module just to override the loading gif?

Reply:
Override is the easiest